_config.ts

```diff
@@ -8,6 +8,8 @@ const site = lume({
   },
 });
 
+site.loadAssets([".css"]);
```

Comment:
Consider looking into [copying static assets](https://lume.land/docs/configuration/copy-static-files/), as we don't do any processing of CSS files. Also, in this case it might be better to be explicit about just doing it to a single file.